app/models/changeset/commit.rb

```diff
@@ -36,6 +36,7 @@ def short_sha
     @data.sha.slice(0, 7)
   end
 
+  # @return [Integer, NilClass]
```

Comment:
```suggestion
  # @return [Integer|NilClass]
```

Reply:
only guidlines I found say to use comma `# @param argname [true, false] only true or false
` https://rubydoc.info/gems/yard/file/docs/GettingStarted.md